# Constants for action
uint8 ACTION_RESET=0
uint8 ACTION_START=1
uint8 ACTION_STOP=2
uint8 ACTION_SET_FRAME_RATE=3
uint8 ACTION_SET_EXTERNAL_TRIGGER=4
uint8 ACTION_SET_EXPOSURE=5

uint8 action
float32 arg1
float32 arg2

---
bool success   # indicate successful run of triggered service
float32 arg1
float32 arg2
string camera
string message # informational, e.g. for error messages
